src/redux/slices/graphSlice.ts

```diff
@@ -2,9 +2,20 @@ import { DSNPUserId } from "@dsnp/sdk/dist/types/core/identifiers";
 import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { GraphChange } from "../../utilities/types";
 
-interface graphState {
-  following: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
-  followers: Record<DSNPUserId, Record<DSNPUserId, boolean>>;
+export enum RelationshipStatus {
+  ESTABlISHED,
+  UPDATING,
```

Comment:
Updating doesn't store the data about if the update is an update to follow or an update to unfollow. Should it?